File: src/main/java/net/frozenblock/linux/mod_compat/FrozenLibIntegration.java
package net.frozenblock.linux.mod_compat;

import net.frozenblock.lib.integration.api.ModIntegration;
import net.frozenblock.linux.util.LinuxConstants;
import net.frozenblock.linux.util.LinuxUtils;

public class FrozenLibIntegration extends ModIntegration {
	public FrozenLibIntegration() {
		super("frozenlib");
	}

	@Override
	public void init() {
		LinuxUtils.log("FrozenLib integration ran!", LinuxConstants.UNSTABLE_LOGGING);
	}
}

File: src/main/java/net/frozenblock/linux/mod_compat/FrozenLibIntegration.kt
package net.frozenblock.linux.mod_compat

import net.frozenblock.lib.integration.api.ModIntegration
import net.frozenblock.linux.util.UNSTABLE_LOGGING
import net.frozenblock.linux.util.log

class FrozenLibIntegration : ModIntegration("frozenlib") {
    override fun init() {
        log("FrozenLib integration ran!", UNSTABLE_LOGGING)
    }
}
